packages/combo-box/test/scrolling.test.js

```diff
@@ -25,30 +25,6 @@ describe('scrolling', () => {
     });
   });
 
-  describe('the height of scroller', () => {
```

Comment:
This test was added in https://github.com/vaadin/vaadin-combo-box/pull/856 and uses hardcoded `116px` value. The `_maxOverlayHeight` method changed in that PR was completely removed in #2497.